packages/lesswrong/components/votes/NamesAttachedReactionsVoteOnComment.tsx

```diff
@@ -380,11 +379,12 @@ const NamesAttachedReactionsCommentBottom = ({
   const reactionsShown = reactionsListToDisplayedNumbers(extendedScore?.reacts ?? null, currentUser?._id);
   const { getAlreadyUsedReactTypesByKarma } = useNamesAttachedReactionsVoting(voteProps)
   const alreadyUsedReactTypesByKarma = getAlreadyUsedReactTypesByKarma();
-  const showOverviewButton = alreadyUsedReactTypesByKarma.length > 1 || reactionsShown.length < alreadyUsedReactTypesByKarma.length;
+
+  const reactionsNotShown = alreadyUsedReactTypesByKarma.filter(r => !reactionsShown.find(rShown => rShown.react===r))
```

Comment:
This line seems convoluted, at least hard to read. I haven't thought through how to do it better.

Reply:
Dunno. I changed the variable names to maybe be more clear. The code kinda needs to be approximately what it is, and given that the variable name should make it clear what's happening I don't think it's worth breaking the function into multiple lines to make it clearer, given that it's a pretty simple function.

Reply:
Ah, so basically just want the difference: alreadyUsedReacts - shownReacts.

Use lodash set operation?

```const reactionsNotShown = _.difference(alreadyUsedReactTypesByKarma, reactionsShownTypes);```

Reply:
It requires an additional step because they aren't actually the same type. 

I made an update adding the additional step. Not sure it's obviously better but there it is.

Reply:
Cool, well probably not worth more effort then